webhook-specification/v1.0/connections.yaml

```diff
@@ -59,57 +74,72 @@ components:
           example: 10
         connections:
           $ref: '#/components/schemas/Connections'
-        followers:
-          $ref: '#/components/schemas/Followers'
         graphKeypair:
           $ref: '#/components/schemas/GraphKeyPair'
     Connections:
-      type: array
-      items:
-        type: object
-        properties:
-          DSNPId:
-            type: integer
-            format: int64
-            example: 1024
-          privacy:
-            type: string
-            example: "Public"
-            enum:
-              - Public
-              - Private
-          connectionType:
-            type: string
-            example: "Follow"
-            enum:
-              - Follow
-              - Friendship
+      type: object
+      properties:
+        data:
+          type: array
+          items:
+            $ref: '#/components/schemas/Connection'
+        pageNumber:
+          type: integer
+        pageSize:
+          type: integer
+        total:
+          type: integer
```

Comment:
Similar to above; I was thinking `pageNumber` and `pageSize` would be properties of an optional `pagination` object, but `totalConnections` should be at the top level (where it is currently).

Reply:
Reasoning: if we don't specify a pagination object in the request, the endpoint can return all connections and omit the pagination object in the response.

Reply:
I like it